raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -1233,7 +1238,7 @@ static void verifyLeaderChangeMessage(
         LeaderChangeMessage leaderChangeMessage = ControlRecordUtils.deserializeLeaderChangeMessage(recordValue);
         assertEquals(leaderId, leaderChangeMessage.leaderId());
         assertEquals(voters.stream().map(voterId -> new Voter().setVoterId(voterId)).collect(Collectors.toList()),
-            leaderChangeMessage.voters());
+            leaderChangeMessage.voters().stream().sorted(Comparator.comparingInt(Voter::voterId)).collect(Collectors.toList()));
```

Comment:
Same here let's fix the formatting a bit. How about:
```java
        assertEquals(
            voters.stream().map(voterId -> new Voter().setVoterId(voterId)),
            leaderChangeMessage.voters().stream().sorted(Comparator.comparingInt(Voter::voterId))
        );
```

Reply:
thanks for the feedback, we still need to collect to List, otherwise the assertion will fail. Updated format in latest commit